### PR TITLE
Create controller not using gem

### PR DESCRIPTION
**Pull request by CamNguyen97**

### CODE REVIEW

**Comment by CamNguyen97** [comment]
ready

**Comment by CamNguyen97** [comment]
ready

**Comment by CamNguyen97** [comment]
ready

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>asdas</h1>
```
Comment (thangtx88) [inline app/views/home/index.html.erb]: cái gì đây

**Comment by thangtx88** [comment]
Commented

**Comment by CamNguyen97** [comment]
ready

**Comment by thangtx88** [comment]
Merged

**Pull request merged**